src/TitleDetails/TitleInformation/TitleInformation.js

```diff
@@ -145,7 +145,7 @@ const TitleInformation = ({
           md={3}
         >
           <KeyValue
-            label={<FormattedMessage id="ui-receiving.title.expectedDate" />}
+            label={<FormattedMessage id="ui-receiving.title.expectedReceiptDate" />}
             value={<FolioFormattedTime dateString={receiptDate} />}
```

Comment:
Why it's FolioFormattedTime ? if it's a date

Reply:
Fixed